Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -9,6 +9,58 @@
  * Refer to the COPYING file of the official project for license.
  *****************************************************************************/
 
+enum ActionSheetCellAccessoryType: Equatable {
+    case none
+    case checkmark
+    case disclosureChevron
+    case custom(image: UIImage)
+    
+    static func == (lhs: ActionSheetCellAccessoryType, rhs: ActionSheetCellAccessoryType) -> Bool {
+        switch lhs {
+        case .checkmark:
+            switch rhs {
+            case .checkmark:
+                return true
+            default:
+                return false
+            }
+        case .disclosureChevron:
+            switch rhs {
+            case .disclosureChevron:
+                return true
+            default:
+                return false
+            }
+        case .custom( _):
+            switch rhs {
+            case .custom( _):
+                return true
+            default:
+                return false
+            }
+        case .none:
+            switch rhs {
+            case .none:
+                return true
+            default:
+                return false
+            }
+        default:
+            assertionFailure("Unhandled ActionSheetAccessoryType")
+        }
+    }
+}
+
+extension UIImage {
+    class var checkmark: UIImage? {
+        return UIImage(named: "checkmark")?.withRenderingMode(.alwaysTemplate)
+    }
+    
+    class var disclosureChevron: UIImage? {
+        return UIImage(named: "disclosureChevron")?.withRenderingMode(.alwaysTemplate)
+    }
+}
```

Comment:
this is a project wide extension of UIImage. I feel a little bit uncomfortable with this, can we somehow limit this to this class and the classes that interact with it?

Reply:
I removed it in the end and just used regular UIImage(named:)